docs/sql-data-sources-hive-tables.md

```diff
@@ -130,7 +130,7 @@ The following options can be used to configure the version of Hive that is used
     <td><code>1.2.1</code></td>
     <td>
       Version of the Hive metastore. Available
-      options are <code>0.12.0</code> through <code>2.3.5</code> and <code>3.1.0</code> through <code>3.1.1</code>.
+      options are <code>0.12.0</code> through <code>3.1.1</code>.
```

Comment:
why is `2.3.5` removed?

Reply:
I ask because from the stability standpoint, I believe 2.3.x line is still considered more stable, and we should explicitly call this out.

Reply:
let us keep it.

Reply:
Done